var/spack/repos/builtin/packages/py-pyfftw/package.py

```diff
@@ -13,11 +13,13 @@ class PyPyfftw(PythonPackage):
     homepage = "http://hgomersall.github.com/pyFFTW"
     url      = "https://pypi.io/packages/source/p/pyFFTW/pyFFTW-0.10.4.tar.gz"
 
+    version('0.12.0', sha256='60988e823ca75808a26fd79d88dbae1de3699e72a293f812aa4534f8a0a58cb0')
     version('0.11.1', sha256='05ea28dede4c3aaaf5c66f56eb0f71849d0d50f5bc0f53ca0ffa69534af14926')
     version('0.10.4', sha256='739b436b7c0aeddf99a48749380260364d2dc027cf1d5f63dafb5f50068ede1a')
 
     depends_on('fftw')
-    depends_on('py-setuptools',    type='build')
-    depends_on('py-cython',        type='build')
-    depends_on('py-numpy@1.6:',    type=('build', 'run'))
-    depends_on('py-scipy@0.12.0:', type=('build', 'run'))
+    depends_on('py-setuptools',        type='build')
+    depends_on('py-cython@0.29:0.999', type='build')
```

Comment:
I would keep this open since cython is still using beta-version scheme and technically each minor release is allowed to break. So we cannot anticipate much.
```suggestion
    depends_on('py-cython@0.29:', type='build')
```

Reply:
Sorry! And thanks for catching this @ax3l .

Reply:
If @adamjstewart agrees, then I'll add that commit. Thanks @ax3l !

Reply:
I actually disagree with this proposed change. If cython releases a 1.X release, this will break the package, as `py-pyfftw` explicitly uses `cython>=0.29, <1.0` in its `setup.py`. Yes, 1.X would probably work for the package, but `setup_requires` would say that the dependency is not satisfied and try to download an older copy of cython, which would break builds on air-gapped networks.

Reply:
Oh I'm sorry, I didn't know that. I will keep then the line as is.
Thank you for your reply @adamjstewart !

Reply:
No problem, thanks Adam! :)